web_src/js/features/repo-issue-list.js

```diff
@@ -194,6 +194,10 @@ function initArchivedLabelFilter() {
   const url = new URL(window.location.href);
   const archivedLabels = document.querySelectorAll('[data-is-archived]');
 
+  if (!archivedLabels.length) {
+    hideElem($('.archived-label-filter'));
```

Comment:
Isn't that a `toggleElem`?

Reply:
No we want to hide the checkbox so, and page is reloading there is no state saved on client side thats why if any label is archived it will automatically show.

Reply:
Yes, but that doesn't change the fact that it is equal to the much simpler `toggleElem('.archived-label-filter', archivedLabels.length)`

Reply:
Ah wait, there's a `return` below.
Okay, then this acceptable.